db/migrations/2022083100590000-remove-unique-addres-to-tokens.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    await queryInterface.changeColumn('tokens', 'address', {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: false
+      });
+    await queryInterface.removeConstraint('tokens', 'tokens_address_key')
+  },
+
+  async down (queryInterface, Sequelize) {
```

Comment:
aren't we missing "addConstraints" ?

Reply:
this table was created as address unique, I had to remove it to use isTransactional logic